XmlSchemas/ConfigurationOverrides/overrides.xsd

```diff
@@ -154,51 +333,162 @@
+                          </xsd:annotation>
+                        </xsd:attribute>
+                        <xsd:attribute name="LegacyAlias" type="xsd:string" use="optional">
+                          <xsd:annotation>
+                            <xsd:documentation>ADVANCED CONFIGURATION. Alias for this cmdlet. This is used to maintain backward compatibility when renaming cmdlets. If more than one alias is required for a single cmdlet, use the service-wide LegacyAliases.</xsd:documentation>
+                          </xsd:annotation>
+                        </xsd:attribute>
+                        <xsd:attribute name="LegacyPaginationCountParameter" type="xsd:string" use="optional">
+                          <xsd:annotation>
+                            <xsd:documentation>ADVANCED CONFIGURATION. This effects only the AWSPowerShell and AWSPowerShell.NetCore modules and applies only to cmdlets configured with LegacyPagination="UseEmitLimit". This attribute specifies where the number of returned items can be read in the response object. This is very rarely used.</xsd:documentation>
+                          </xsd:annotation>
+                        </xsd:attribute>
+                        <xsd:attribute name="OutputWrapper" type="xsd:string" use="optional">
+                          <xsd:annotation>
+                            <xsd:documentation>ADVANCED CONFIGURATION. This is used for services that have an extra encapsulation layer of the response object. This specifies the name of the property of the response object representing the actual service response. Because OutputProperty supports descendant properties, configuring OutputWrapper is only needed for paginated operations (AutoIterate doesn't support nested properties).</xsd:documentation>
+                          </xsd:annotation>
+                        </xsd:attribute>
+                        <xsd:attribute name="Exclude" type="xsd:boolean" use="optional">
+                          <xsd:annotation>
+                            <xsd:documentation>ADVANCED CONFIGURATION. Defaults to false. When set to true, a cmdlet won't be generated for this operation. This is normally used only when an handwritten ("advanced") cmdlet is created instead.</xsd:documentation>
+                          </xsd:annotation>
+                        </xsd:attribute>
+                        <xsd:attribute name="Remove" type="xsd:boolean" use="optional">
+                          <xsd:annotation>
+                            <xsd:documentation>ADVANCED CONFIGURATION. Defaults to false. This attribute is not part of the service configuration. This is used only as part of the configuration override to remove an existing cmdlet from the existing service configuration (this is needed if an operation is removed from the AWS SDK for .NET).</xsd:documentation>
```

Comment:
and is needed if an operation is removed from the AWS SDK for .NET

Reply:
ok